Makefile

```diff
@@ -101,9 +101,6 @@ fmt: ## Run go fmt against code.
 vet: ## Run go vet against code.
 	go vet ./...
 
-.PHONY: test
-test: manifests generate fmt vet envtest ## Run tests.
```

Comment:
How should I run tests locally after this change?

Reply:
We still have the `test-unit`directive.